.eslintrc.json

```diff
@@ -45,6 +45,25 @@
         "ignoreRestSiblings": true
       }
     ],
+    "@typescript-eslint/member-ordering": [
+      "error",
+      {
+        "default": {
+          "memberTypes": [
+            "static-method",
+            "static-field",
+            "abstract-method",
+            "abstract-field",
+            "instance-field",
+            "constructor",
+            "public-instance-method",
+            "protected-instance-method",
+            "private-instance-method"
```

Comment:
Which of the [supported options](https://typescript-eslint.io/rules/member-ordering/#all-supported-options) is this? (Are we missing getters and setters here?)